.github/workflows/integration-mariadb.yml

```diff
@@ -147,7 +147,6 @@ jobs:
         run: |
           mkdir data
           ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
-          ./occ config:system:set debug --value=true --type=boolean
           ./occ config:system:set hashing_default_password --value=true --type=boolean
```

Comment:
that being said, we should copy this to the script as well, so locally uses the fast hashing as well

Reply:
and the federated server

Reply:
Added in a new commit.